### PR TITLE
Add note about MS Edge requirements

### PR DESCRIPTION
**Pull request by AndreyTretyak**
MS Edge needs to be started at least once for MS Intune to work properly. This took me some time to figure it out, so adding it in the docs to save other troubles. Feel free to change my wording if there is a better way to express this.

### CODE REVIEW

File: memdocs/intune/user-help/enroll-device-linux.md [hunk 1]
```diff
@@ -56,10 +56,11 @@ Install these apps on your device prior to enrollment:
 ## Enroll device  
 Follow these steps to register your personal Linux device on your organization's network.  
 
-1. Open the Microsoft Intune app.  
-2. Sign in with your work or school account.    
-3. Review the pre-enrollment screens. Then select **Next** to begin enrollment. 
-4. Wait a few minutes while the Intune app enrolls your device. 
+1. Open Microsoft Edge. It needs to run at least once after install to initalize componetns that would be used by Microsoft Intune.
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/enroll-device-linux.md:59]:
```suggestion
1. Open Microsoft Edge. It needs to run at least once after installation to initialize components that would be used by Microsoft Intune.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AndreyTretyak, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks.

**Comment by tiburd** [comment]
@lenewsad 
Hi Laura,
Can you please review these changes?
Thanks!

**Comment by lenewsad** [comment]
Hi @yogkumgit @tiburd @AndreyTretyak, waiting for PM review.

**Comment by benferse** [comment]
Starting Edge is not a requirement for the Intune app to function properly. We have test environments for Intune that don't even install Edge. It sounds like there is something else specific to your environment @AndreyTretyak that is going on - if this is reproducible can you contact support to investigate further?

**Comment by AndreyTretyak** [comment]
> Starting Edge is not a requirement for the Intune app to function properly. We have test environments for Intune that don't even install Edge. It sounds like there is something else specific to your environment @AndreyTretyak Andrey Tretyak FTE that is going on - if this is reproducible can you contact support to investigate further?

@benferse  thank you for the response. 
Just out of curiosity, have you tried it with accounts that require 2FA? In my case three separate Ubuntu VMs (22 and 20) were showing blank windows instead of 2FA screen until I've installed and started MS Edge.
What would be the best way to contact support about this?

**Comment by lenewsad** [comment]
Hi @AndreyTretyak, There are a couple of support options available: 

- Post to [Microsoft Intune forums](https://docs.microsoft.com/answers/products/mem) on Microsoft Q & A. This space is moderated by our community MVPs who are very experienced, knowledgeable, and helpful. 

- Use [Microsoft Support for business](https://aka.ms/cmcbsupport) to search for an issue, or open a case with professional support. 

- Contact your Microsoft Premier team, such as a Premier Field Engineer or Technical Account Manager. 

  FYI @benferse

**Comment by lenewsad** [comment]
Thanks again @AndreyTretyak for reaching out to the Docs team. I'm going to close this PR since we determined that no changes are needed. Please don't hesitate to open more PRs with us in the future--we're happy to review them.

**Pull request closed without merging**